README.md

```diff
@@ -24,8 +24,7 @@ The [master](https://github.com/butterproject/butter-desktop) branch which conta
 
 #### Quickstart:
 
-1. `npm install -g grunt-cli bower`
```

Comment:
Alright ill add them to the pre start script :smile:





On Thu, Mar 10, 2016 at 6:53 AM -0800, "Jean van Kasteel" <notifications@github.com> wrote:





> @@ -24,8 +24,7 @@ The [master](https://github.com/butterproject/butter-desktop) branch which conta
>
>  #### Quickstart:
>
> -1. `npm install -g grunt-cli bower`

and I'd add, we even still need grunt & bower _globally_ ^^

---
Reply to this email directly or view it on GitHub:
https://github.com/butterproject/butter-desktop/pull/314/files#r55689836